functions/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "functions",
```

Comment:
Could name this something else and mark it as `"private": true` just for good measure.